src/tasks.py

```diff
@@ -46,6 +98,12 @@ def add_bpm(x):
         bpm, confidence = get_song_bpm(song)
 
         x['BPM'] = dict({'value': bpm, 'confidence': confidence})
+
+        if x['DB_EXISTS']:
```

Comment:
We might want to extract this into a separate function to avoid code duplication

Reply:
Done :)
Has been moved to a _save_to_db function